pkg/controllers/vdb/revivedb_reconciler.go

```diff
@@ -80,6 +80,11 @@ func (r *ReviveDBReconciler) Reconcile(ctx context.Context, _ *ctrl.Request) (ct
 		return ctrl.Result{}, nil
 	}
 
+	// Check if restoring from a restore point is supported
+	if err := r.isRestoreSupported(); r.Vdb.IsRestoreConfig() && err != nil {
```

Comment:
Should we revert the check? Check if its a restore, then validate that we support it. Otherwise, we will always generate the event message and error, but can choose to ignore it if not doing a restore.